packages/epics/src/treasury/components/assets/assets-section.tsx

```diff
@@ -46,7 +48,12 @@ export const AssetsSection: FC<AssetSectionProps> = () => {
         tabs={filterOptions}
       />
       {Array.from({ length: pages }).map((_, index) => (
-        <AssetsList page={index + 1} key={index} activeFilter={activeFilter} />
+        <AssetsList
+          basePath={basePath}
+          page={index + 1}
+          key={index}
```

Comment:
using `index` as key is an anti-pattern and generally not good (with exceptions) [read more](https://dev.to/shiv1998/why-not-to-use-index-as-key-in-react-lists-practical-example-3e66)